aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -598,6 +598,11 @@ class QConvInt8 final {
           reinterpret_cast<uint8_t*>(qnnp_w_data),
           reinterpret_cast<int32_t*>(bias.data_ptr<c10::qint32>()));
       pack_w = pack_data.w.get();
+      if (at::globalContext().releaseOriginalWeights()) {
+        // On mobile, we release the original weight by resetting the intrusive_ptr.
+        // Calling unpack after this will throw an assertion.
+        pack_data.orig_weight.reset();
```

Comment:
Is this sufficient btw to release underlying memory?

Reply:
Yeah, if there are no additional references to the weight (which I confirmed that JIT doesn't hold on to one), it will free up memory.